src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer.java

```diff
@@ -251,6 +252,26 @@ public String getName() {
         return name;
     }
 
+    /**
+     * Get the prefix for fetching all project names.
+     *
+     * @return projects prefix.
+     */
+    @Exported
+    public String getProjectPrefix() {
+        return projectPrefix;
+    }
+
+    /**
+     * Set the prefix for fetching all project names.
+     *
+     * @param projectPrefix prefix of the projects to list from the Gerrit Server.
+     */
+    @Exported
+    public void setProjectPrefix(String projectPrefix) {
```

Comment:
Maybe I'm missing something, but where does this get called?

If it's not called from anywhere, the projectPrefix uninitialized which results in the full list being returned, i.e. the same behavior as before.

Reply:
For backward compatibility, it will work as before.

Need to find a place to hook into it, do you have any suggestions? I’m not very familiar with this codebase :-(

Reply:
Yes this uses an outdated way of doing Jenkins form submission. See `public void doConfigSubmit(StaplerRequest req, StaplerResponse rsp) throws ServletException...` in the decriptor.